lib/utils.js

```diff
@@ -392,6 +392,29 @@ var Utils = module.exports = {
 
   validateParameter: function(value, expectation, options) {
     return ParameterValidator.check(value, expectation, options);
+  },
+
+  formatReferences: function (obj) {
+    var references = {};
+
+    if (!obj || !_.isPlainObject(obj) || !obj.references) {
+      return obj;
+    }
+
+    if (_.isPlainObject(obj.references)) {
+      references = obj.references;
+    } else {
+      deprecate('Non-object references property. Expected { references: { model: "value" } } instead of { references: "value" }.');
+      references.model = obj.references;
```

Comment:
Just change the deprecation message above to:
```js
deprecate('Non-object references property. Expected { references: { model: "value", key: "key" } } instead of { references: "value", referencesKey: "key", ... }.');
```